### PR TITLE
🐛 Remove the OPENEXR_INSTALL_EXAMPLES CMake option

### PR DESCRIPTION
**Pull request by Jamim**
_Hello @cary-ilm,_

The `OPENEXR_INSTALL_EXAMPLES` option is misleading and doesn't have any effect.
It might be safely removed in favor of the `OPENEXR_BUILD_EXAMPLES` option.

_Best regards!_

### CODE REVIEW

File: website/install.rst [hunk 1]
```diff
@@ -460,7 +460,7 @@ Component Options
   Install the binary programs (exrheader, exrinfo,
   exrmakepreview, etc). Default is ``ON``.
   
-* ``OPENEXR_INSTALL_EXAMPLES``
+* ``OPENEXR_BUILD_EXAMPLES``
 
   Build and install the example code. Default is ``ON``.
```
Comment (cary-ilm) [inline website/install.rst]: Should remove any reference to install here:
```suggestion
  Build the example code. Default is ``ON``.
```

**Review by cary-ilm — CHANGES_REQUESTED** [review]
One small suggestion below, but otherwise perfect, thanks!  Last time I looked I suspected that was superfluous but didn't do anything about it.

File: website/install.rst (hunk 1 above)
Comment (Jamim) [reply website/install.rst]: Thank you! I've submitted an updated commit.

**Review by cary-ilm — APPROVED** [review]
Thanks!

**Pull request merged**